.gitignore

```diff
@@ -8,6 +8,7 @@ __pycache__
 
 # Build + installation data
 build/
+third-party/
```

Comment:
Is third-party/ used to store generated sycl kernels?  What will the directory structure be under third-party?